src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Syntax/SyntaxListBuilderOfT.cs

```diff
@@ -25,21 +23,9 @@ internal SyntaxListBuilder(SyntaxListBuilder builder)
         _builder = builder;
     }
 
-    public bool IsNull
-    {
-        get
-        {
-            return _builder == null;
-        }
-    }
+    public bool IsNull => _builder == null;
```

Comment:
>IsNull [](http://example.com/codeflow?start=16&length=6)

How does this get null?

Reply:
It's a struct, like `ImmutableArray<T>`. So, it can be null just be having code like `SyntaxListBuilder<SyntaxNode> builder = default`. FYI that this type was copied long ago from Roslyn: https://sourceroslyn.io/#Microsoft.CodeAnalysis/Syntax/InternalSyntax/SyntaxListBuilder%25601.cs,b46872cbff74411b,references.

Reply:
Might need more coffee, isn't SyntaxListBuilder (not SyntaxListBuilder<TNode>) a class?

Reply:
Never mind, more coffee helped!